src/bug_report.rs

```diff
@@ -0,0 +1,209 @@
+use crate::utils::exec_cmd;
+use reqwest;
+use std::fs;
+use std::path::PathBuf;
+
+const GIT_IO_BASE_URL: &str = "https://git.io/";
+
+pub fn create() {
+    let os_info = os_info::get();
+
+    let environment = Environment {
+        os_type: os_info.os_type(),
+        os_version: os_info.version().to_owned(),
+        shell_info: get_shell_info(),
+        starship_config: get_starship_config(),
+    };
+
+    let link = make_github_issue_link(crate_version!(), environment);
+
+    if open::that(&link).is_ok() {
+        print!("Take a look at your browser. A GitHub issue has been populated with your configuration")
+    } else {
+        let link = reqwest::Client::new()
+            .post(&format!("{}{}", GIT_IO_BASE_URL, "create"))
+            .form(&[("url", &link)])
+            .send()
+            .and_then(|mut response| response.text())
+            .map(|slug| format!("{}{}", GIT_IO_BASE_URL, slug))
+            .unwrap_or(link);
+
+        println!(
+            "I was unable to launch your browser. You'll have to copy this link instead:\n\n  {}",
```

Comment:
I'd rather we don't call attention to being unable to launch the browser:
```suggestion
            "Click this link to create a GitHub issue populated with your configuration:\n\n  {}",
```